src/rgw/rgw_sal_motr.h

```diff
@@ -921,6 +922,8 @@ class MotrMultipartUpload : public MultipartUpload {
        }
   virtual ~MotrMultipartUpload() = default;
 
+  void set_version_id(std::string version_id) { _version_id = version_id; };
```

Comment:
I would recommend to use `_version_id` name for the function argument and `version_id` for the actual class variable.

Reply:
done